src/sage_docbuild/conf.py

```diff
@@ -248,7 +248,7 @@ def set_intersphinx_mappings(app, config):
         if os.path.isdir(os.path.join(invpath, directory)):
             src = os.path.join(refpath, directory)
             dst = os.path.join(invpath, directory, 'objects.inv')
-            app.config.intersphinx_mapping[src] = dst
+            app.config.intersphinx_mapping[directory] = (src, dst)
 
```

Comment:
`directory` is used as a name for `intersphinx_mapping`? Then perhaps you may want to edit the comment starting from L240.

Reply:
It is supposed to be a unique identifier, this seemed like the most natural choice.